lib/configuration.rb

```diff
@@ -268,7 +276,7 @@ def setup_single_compiler(compiler, is_release)
     compiler[:cmake_extra_flags] = setup_compiler_extra_flags(compiler, is_release)
     compiler[:num_parallel_builds] = setup_compiler_num_processors(compiler)
 
-    raise CannotMatchCompiler, 'Decent CI currently only deployed with Visual Studio version 16 (2019)' if compiler[:name] =~ /.*Visual Studio.*/i && compiler[:version] != 16
+    raise CannotMatchCompiler, 'Decent CI currently only deployed with Visual Studio version 16 (2019) and 17 (2022)' if compiler[:name] =~ /.*Visual Studio.*/i && !compiler[:version].in?([16, 17])
```

Comment:
I think I may need to still tweak something on Decent to make sure we **skip** a different VS version and not **fail** on it.